manifest.php

```diff
@@ -27,7 +27,7 @@
     'label' => 'LTI Delivery Tool Provider',
     'description' => 'The LTI Delivery Tool Provider allows third party applications to embed deliveries created in Tao',
     'license' => 'GPL-2.0',
-    'version' => '7.0.0',
+    'version' => '7.1.0',
```

Comment:
Is this really should be a minor version bump? IMO, looks more like a patch version bump for me.